lib/opentok/connections.rb

```diff
@@ -0,0 +1,32 @@
+module OpenTok
+  #  A class for working with OpenTok signconnectionsals.
```

Comment:
Minor typo error "signconnectionsals"

Reply:
@JayTokBox please take a look at the comment and also fix the merge conflict, thanks!